packages/nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -266,28 +272,6 @@ export class PersonalizeMiddleware extends MiddlewareBase {
     };
   }
 
-  protected getExperienceParams(req: NextRequest): ExperienceParams {
```

Comment:
Hold on, this method was already a protected on, not private.  User could override it.

Reply:
And making it public is not correct either, PersonalizeMiddleware like any other middlware is in fact a command pattern and should have just a single public method -> to execute the middleware